test/e2e/horizontal_pod_autoscaling.go

```diff
@@ -32,29 +33,44 @@ const (
 )
 
 var _ = Describe("Horizontal pod autoscaling (scale resource: CPU) [Skipped]", func() {
-	var rc *ResourceConsumer
-	f := NewFramework("horizontal-pod-autoscaling")
 
 	titleUp := "Should scale from 1 pod to 3 pods and from 3 to 5"
 	titleDown := "Should scale from 5 pods to 3 pods and from 3 to 1"
+	titleMult := "Should HPA scale in parallel"
```

Comment:
nit: why is this phrased differently from the others? HPA seems redundant, and having the counts seems helpful.